src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -36,21 +39,36 @@
   private static final Logger LOGGER = Logger.getLogger(KubectlWrapper.class.getName());
   private static final String CHARSET = "UTF-8";
 
+  private JenkinsRunContext context;
+  private KubeConfig kubeConfig;
+
   /**
-   * Runs the specified kubectl command within the specified {@link JenkinsRunContext}'s
-   * environment.
+   * Create a non-static reference of the wrapper to encapsulate the JenkinsRunContext and
+   * KubeConfig
    *
-   * @param context The {@link JenkinsRunContext} whose environment the command will execute in.
+   * @param context The {@link JenkinsRunContext} the jenkins context environment the command will
+   *     execute in.
    * @param kubeConfig The {@link KubeConfig} containing the credentials for the cluster being
    *     executed against.
+   */
+  public KubectlWrapper(JenkinsRunContext context, KubeConfig kubeConfig) {
+    this.context = context;
+    this.kubeConfig = kubeConfig;
+  }
+  /**
+   * Runs the specified kubectl command within the specified {@link JenkinsRunContext}'s
+   * environment.
+   *
    * @param command The kubectl command to be run.
    * @param args Arguments for the command.
    * @throws IOException If an error occurred while executing the command.
    * @throws InterruptedException If an error occured while executing the command.
+   *     <p>TODO: make a non-static version of this that stores the arguments such that it can be
```

Comment:
I think this TODO can be removed since that's what you appear to have done here.